rpm/maliit-framework-wayland.spec

```diff
@@ -109,6 +109,7 @@ sed -i -e 's:@PATH@:%{_libdir}/maliit-framework-tests:g' %{buildroot}/opt/tests/
 %license LICENSE.LGPL
 %{_bindir}/maliit-server
 %{_libdir}/libmaliit-plugins.so.*
+%{_libdir}/libmaliit-glib.so*
```

Comment:
I wouldn't necessarily want to have this in the main package.

Reply:
added it to it's own package by doing `%files glib`